src/tiled/mainwindow.cpp

```diff
@@ -1773,3 +1777,25 @@ void MainWindow::reloadError(const QString &error)
 {
     QMessageBox::critical(this, tr("Error Reloading Map"), error);
 }
+
+void MainWindow::setUseDarkTheme(bool useDarkTheme)
+{
+    if (useDarkTheme)
+    {
+        QFile darkThemeFile(QLatin1String(":darkstyle/style.qss"));
+        if (darkThemeFile.exists())
+        {
+            darkThemeFile.open(QFile::ReadOnly | QFile::Text);
```

Comment:
Rather than doing explicit `exists` check and then trying to open the file, you should just call `open` and check its return value:
```
if (darkThemeFile.open(QFile::ReadOnly | QFile::Text)) {
```